cve_bin_tool/checkers/icu.py

```diff
@@ -12,11 +12,16 @@
 
 
 class IcuChecker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = [r"icu", r"international_components_for_unicode"]
+    CONTAINS_PATTERNS = [
+        r"-i or --icudatadir       directory for locating any needed intermediate data files,",
+        r"-j or --write-java       write a Java ListResourceBundle for ICU4J, followed by optional encoding",
+        r"failed to load root collator \(ucadata.icu\) - %s",
+        r"is about 300kB larger than the ucadata-implicithan\.icu version\.",
+        r"the ucadata-unihan\.icu version of the collation root data",
+    ]
+    FILENAME_PATTERNS = [r"genrb", r"uconv"]
     VERSION_PATTERNS = [
-        r"icu[_-]([0-9]+\.[0-9]+\.[0-9]+)",
-        r"ICU ([0-9]+\.[0-9]+\.[0-9]+)",
-        r"icu[_-][relas-]*((0*(?:[1-6][0-9]?))+(\-[0-9]+)*)",
+        r"icu-([0-9]+\.[0-9]+(\.[0-9]+)?)",
+        r"ICU ([0-9]+\.[0-9]+(\.[0-9]+)?)",
```

Comment:
I removed the patterns, because for some reason they were causing an error. Also,
`r"icu[_-][relas-]*((0*(?:[1-6][0-9]?))+(\-[0-9]+)*)",` this was not present in any of the newer or older version